nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -103,18 +103,23 @@ def mean_minmax_statistic_collector(use_per_sample_stats: bool,
     @staticmethod
     def get_weight_tensor(model: onnx.ModelProto, node: NNCFNode) -> Tuple[str, np.ndarray]:
         onnx_graph = ONNXGraph(model)
-        node = onnx_graph.get_node_by_name(node.node_name)
-        return onnx_graph.get_weight_tensor(node)
+        onnx_node = onnx_graph.get_node_by_name(node.node_name)
+        return onnx_graph.get_weight_tensor(onnx_node)
 
     @staticmethod
-    def get_weight_tensor_port_id(model: onnx.ModelProto, node: NNCFNode) -> Optional[int]:
+    def get_weight_tensor_port_id(model: onnx.ModelProto, node: NNCFNode) -> int:
         onnx_graph = ONNXGraph(model)
-        node = onnx_graph.get_node_by_name(node.node_name)
-        weight_tensor_name, _ = onnx_graph.get_weight_tensor(node)
-        for i, input_name in enumerate(node.input):
-            if input_name == weight_tensor_name:
-                return i
-        return None
+        onnx_node = onnx_graph.get_node_by_name(node.node_name)
+        return onnx_graph.get_weight_tensor_port_id(onnx_node)
+
+    @staticmethod
+    def get_weight_tensor_quantization_axis(model: onnx.ModelProto, node: NNCFNode,
```

Comment:
There is no such method in the main backend entity.

Reply:
Done